applications/StructuralMechanicsApplication/custom_elements/spring_damper_element_3D2N.cpp

```diff
@@ -390,8 +395,7 @@ void SpringDamperElement3D2N::ClearNodalForces()
 {
     KRATOS_TRY;
     
-    if( GetGeometry()[0].SolutionStepsDataHas(EXTERNAL_FORCE) && GetGeometry()[0].SolutionStepsDataHas(INTERNAL_FORCE) )
-    {
+    if( GetGeometry()[0].SolutionStepsDataHas(EXTERNAL_FORCE) && GetGeometry()[0].SolutionStepsDataHas(INTERNAL_FORCE) ) {
       array_1d<double, 3 > & ExternalForce = GetGeometry()[0].FastGetSolutionStepValue(EXTERNAL_FORCE);
       array_1d<double, 3 > & InternalForce = GetGeometry()[0].FastGetSolutionStepValue(INTERNAL_FORCE);
 
```

Comment:
guys, this should NOT be done this way.... an element is essentially forbidden to SET something or to write to its nodes!!

Reply:
This element is from @qaumann